### PR TITLE
fix: running tests in CI

### PR DESCRIPTION
**Pull request by mikhailOK**
Make tests run again.
Script should be looking at a different directory for test binaries, most likely broke 
during rust-toolchain update.

Test plan
---------
Check that this is not the case anymore: ========= collected 0 test binaries:

### CODE REVIEW

**Comment by mikhailOK** [comment]
Issues for failing tests: #2366 #2367

**Comment by SkidanovAlex** [comment]
What is being fixed here, and how is it fixed?

**Comment by ailisp** [comment]
> What is being fixed here, and how is it fixed?

Since our last update rust version, test binary path changed and ci was actually not running any test. My fault didn't notice that. Until today Misha noticed and fixed the path

File: chain/jsonrpc/tests/rpc_query.rs [hunk 1]
```diff
@@ -474,7 +474,7 @@ fn test_genesis_config() {
     test_with_client!(client, async move {
         let genesis_config = client.EXPERIMENTAL_genesis_config().await.unwrap();
         assert_eq!(genesis_config["config_version"].as_u64().unwrap(), 1);
-        assert_eq!(genesis_config["protocol_version"].as_u64().unwrap(), 5);
+        assert_eq!(genesis_config["protocol_version"].as_u64().unwrap(), 7);
```
Comment (evgenykuzyakov) [inline chain/jsonrpc/tests/rpc_query.rs:477]: Hmm, how did this pass before?

Comment (ailisp) [reply chain/jsonrpc/tests/rpc_query.rs:477]: It was not run before (after upgrade rust toolchain)

**Pull request merged**